elastic-agent-client.proto

```diff
@@ -4,7 +4,7 @@
 
 syntax = "proto3";
 
-package proto;
+package agentclient;
```

Comment:
You can put a version in this I believe, like `agent.client.v1` and `agent.client.v2`

Reply:
Maybe `agent.control.v1` and v2

Reply:
We shouldn't use client really since this will generate both a client and server